.ipynb_checkpoints/command_analogues/ls.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-a', '--all', help='do not ignore entries starting with .', action='store_true')
+parser.add_argument('path', type=str, nargs='*', default=[os.getcwd()])
+
+args = parser.parse_args()
+
+if args.all:
+	for i in range(len(args.path)):
```

Comment:
Почему не 
```suggestion
	for path in args.path:
```
Так же проще